content/02.introduction.md

```diff
@@ -2,5 +2,94 @@
+Along with co-occurrence, one study used a bi-clustering approach to detect DaG relevant sentences from Pubmed abstracts [@tag:global_network].
+This clustering approach used dependency paths to group similar sentences together.
+We used the results of this work for our domain heuristic label functions.
+The benefit of using the above approaches is inessential need for annotated data.
+These methods provide great recall performance in terms of extraction, but can suffer in precision performance.
+
+Supervised machine learning for DaG has prospered with the creation of hand crafted datasets [@tag:befree; @tag:eu_adr; @tag:comagc; @tag:craft].
+This datasets served as a gold standard for tuning and testing machine learning models in DaG extraction.
+Copious amount of effort used support vector machines to detect DaG relationships [@tag:befree; @tag:dtminer; @tag:ensemble_svm].
+This model performs well in large feature spaces, but perform slowly as the number of data points increases.
+Following support vector machines, effort has focuses on using deep learning models for extraction.
+One such effort used a pre-trained recurrent neural network for DaG extraction [@tag:biobert].
+Similar to our approach another study used distant supervision for DaG extraction [@tag:deep_dive_dag].
+Due to the success of these two models, we take a deep learning approach for our discriminative model.
+
+#### Compound Treats Disease
+
+Effort for Compound-treats-Disease (CtD) extraction focuses on identifying sentences that mention current drug treatments or discovering new uses for existing drugs (drug repurposing).
+One study used an inference model combined with co-occurrence scores to identify sentences regarding drug repurposing predictions [@tag:abc_drugs].
+This approach consisted of using an inference model on previously established drug-gene and gene-disease relationships to infer novel drug-disease interactions.
+Each identified pair was ranked using a co-occurrence score that was calculated using pair mention frequency on Pubmed abstracts [@tag:abc_drugs].
+A similar study used a similar inference model for CtD extraction [@tag:copub_discovery].
+Following co-occurrence one study consisted of using manually curated rules on Pubmed abstracts for CtD extraction [@tag:ctd_medline].
+These rules were based on identifying key drug treatment phrases and wording for extraction. 
+We used these patterns as inspiration for some of our CtD specified label functions.
+Lastly, one study used a  bi-clustering approach to identify sentences relevant to the CtD [@tag:global_network].
+As mentioned for DaG we use the results of this study for domain heuristic label functions.
+
+Majority of supervised machine learning efforts use the Biocreative V  dataset [@tag:biocreative_v].
```

Comment:
```suggestion
Recent work with supervised machine learning methods has often focused on compounds that induce a disease: an important question for toxicology and the subject of the BioCreative V dataset [@tag:biocreative_v].
```

Reply:
You should add a few references to the first part of the sentence since you assert that this is a major focus of supervised work in this area.